scripts/suggestion-bot.js

```diff
@@ -0,0 +1,72 @@
+function getPullRequestNumber() {
```

Comment:
Did you grab the implementation of `suggestion-bot.js` somewhere else? Otherwise it seems like a good candidate for being its own OS project :)

Reply:
Yeah, I've been thinking about extracting it. I'll do that later tonight maybe.

Reply:
https://www.npmjs.com/package/suggestion-bot